modules/soe_paragraph_cta_list/templates/cta-list-paragraph.html.twig

```diff
@@ -0,0 +1,47 @@
+
+{#
+/**
+ * @file
+ * CTA List Paragraph
+ *
+ * Template for the CTA List paragraph type.
+ *
+ * Available variables:
+ *
+ * {{ header }}
+ * {{ deck }}
+ * {{ links }}
+ *
+ */
+#}
+{%- set attributes = attributes.addClass(['su-cta-list-paragraph', modifier_class]) -%}
+{%- set link_count = links.su_cta_list_links['#items']|length -%}
+{# We need to figure out how many links go in each column. #}
+{%- if link_count%2 == 0 -%}
+    {%- set col_1_link_count = link_count/2 -%}
+{%- else -%}
+    {%- set col_1_link_count = ((link_count/2)-(1/2))+1 -%}
+{%- endif -%}
+
+<div{{ attributes }}>
+    {{ header }}
+    {{ deck }}

```

Comment:
If the regions don't have a wrapper around them could they be consolidated into one?

Reply:
consolidated.